wiki/Game_mode/osu!mania/en.md

```diff
@@ -186,133 +186,17 @@ The **[05 March 2015 (2015-03-05) update](https://osu.ppy.sh/home/changelog)** g
 
 ## Scoring
 
-*Scoring values can be found in [osu!mania scoring system](/wiki/Score/osu!mania).*
+[The osu!mania score](/wiki/Gameplay/Score/osu!mania) is capped at 1 million and consists of two parts, each contributing 50% of total score:
 
-Scoring section details all the intricacies of scoring, including mathematical formula.
+1. Base score, which is [judgement](/wiki/Gameplay/Judgement/osu!mania)-centered. Judgement determines an object's base scoring value (MAX, 300, 200, 100, 50, or 0 in case of a miss).
```

Comment:
```suggestion
1. Base score, which is [judgement](/wiki/Gameplay/Judgement/osu!mania)-centred. Judgement determines a hit object's base scoring value (MAX, 300, 200, 100, 50, or 0 in case of a miss).
```

Reply:
> centred

ಠ_ಠ